tools/bazel.rc

```diff
@@ -79,10 +79,6 @@ build:fuzzer_asan --action_env=ASAN_OPTIONS=detect_leaks=1:color=always
 build:fuzzer_asan --action_env=LSAN_OPTIONS=suppressions=test/core/test_util/lsan_suppressions.txt:report_objects=1
 build:fuzzer_asan --action_env=ASAN_OPTIONS=detect_odr_violation=0  # https://github.com/google/sanitizers/issues/1017
 
-build:coverage --copt=-fprofile-instr-generate
```

Comment:
Are the changes to this file intentional?

Reply:
accident, fixed.